dbscripts/xml/upgrade.xml

```diff
@@ -162,6 +171,8 @@
 		<code function="installEmailTemplate" key="EDITOR_DECISION_REVERT_DECLINE" locales="en_US" />
 		<code function="installEmailTemplate" key="EDITOR_DECISION_REVERT_INITIAL_DECLINE" locales="en_US" />
 		<code function="installEmailTemplate" key="EDITOR_DECISION_SKIP_REVIEW" locales="en_US" />
+		<code function="installEmailTemplate" key="EDITOR_DECISION_BACK_TO_SUBMISSION_NOTIFY_REVIEWER" locales="en_US" />
+		<code function="installEmailTemplate" key="EDITOR_DECISION_BACK_TO_PREVIOUS_EXTERNAL_REVIEW_ROUND_NOTIFY_REVIEWER" locales="en_US" />
```

Comment:
I think maybe these are not all of the new email templates, but we can revisit this once the mailables have been adjusted.

Reply:
modified and removed .